### PR TITLE
fix(IconButton): add borderRadius to button's overlay

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-1879)

### CODE REVIEW

File: src/icon-button.css.ts [hunk 1]
```diff
@@ -417,6 +417,7 @@ export const overlay = style({
     width: '100%',
     height: '100%',
     backgroundColor: 'transparent',
+    borderRadius: '50%',
```
Comment (marcoskolodny) [inline src/icon-button.css.ts:420]: I have no idea why the overlay was rounded in most cases, but not in Popover (only in iOS).

This fixes the issue

File: src/tooltip.tsx [hunk 2]
```diff
@@ -569,7 +569,7 @@ export const BaseTooltip: React.FC<BaseTooltipProps> = ({
 
                                                 {onClose && (
                                                     <div className={styles.closeButtonIcon}>
-                                                        <InternalIconButton
+                                                        <IconButton
```
Comment (marcoskolodny) [inline src/tooltip.tsx:572]: Not related. We can use IconButton here, we are not using hasOverlay/isOverMedia in here

**Review by yceballost — APPROVED** [review]
🫶

**Review by atabel — APPROVED** [review]

**Pull request merged**